### ISSUE TITLE
Some skins may not work

### ISSUE THREAD

**Issue by mrmelon54** [PolyMarsDev/Minecraft-Skin-Grabber#1]
When uploading skins the player can choose between slim and thick arms so the texture having a transparent pixel doesn't matter. You can draw anything you like outside of the mapped parts of the texture.

**Comment by BunnyTub** [comment]
I'm confused.

**Comment by mrmelon54** [comment]
You can draw anything you like in the image file outside the actual skin so the pixel next to the arm might not be transparent for a slim arms texture

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mandar1jn** [comment]
Yes, but you can disable the outer layer in the minecraft settings. Just see this as a skin viewer without the outer layer enabled

**Comment by mrmelon54** [comment]
That's not what I was talking about. I meant if the image file had some pixels outside of where the skin is. e.g. the top left corner somewhere that minecraft doesn't use for the final skin once it has been UV mapped. I have seen skin authors draw initials in places like this. The pixel I was refering to was the one that is just to the right of the arm which is being used to detect if its an alex player model or not. The problem is that this pixel might not be transparent if the author added extra data outside of the skin (e.g. initials)

**Comment by byemc** [comment]
> That's not what I was talking about. I meant if the image file had some pixels outside of where the skin is. e.g. the top left corner somewhere that minecraft doesn't use for the final skin once it has been UV mapped. I have seen skin authors draw initials in places like this. The pixel I was refering to was the one that is just to the right of the arm which is being used to detect if its an alex player model or not. The problem is that this pixel might not be transparent if the author added extra data outside of the skin (e.g. initials)

This is unlikey to happen, this is an edge case. Poly doesn't actually *need* to fix it, and it would probably take a lot of his time to fix, which might not be appealing (he also wrote the code in 2019, so he mightn't figure out how it works ever again)

**Comment by mrmelon54** [comment]
well its still an issue if its going to be fixed or not

so I'll just leave this open incase someone needs to know about it
